.github/PULL_REQUEST_TEMPLATE.MD

```diff
@@ -38,3 +50,5 @@ Fixes #123 and [JIRA-0011](https://bigbite.atlassian.net/browse/JIRA-0011) - We'
 - [ ] I have made corresponding changes to the documentation
 - [ ] My changes generate no new warnings
 - [ ] Any dependent changes have been merged and published in downstream modules
+- [ ] I have disclosed all areas where AI assistance was used when creating this pull request, if applicable
+- [ ] AI code suggestions were reviewed and modified to match our coding standards
```

Comment:
To me, this one reads as though the expectation is that AI "contributions" pass linter checks. Am I reading that correctly?

Reply:
Yeah i think this should say something like "AI Code suggestions have been manually reviewed and I understand how it works" (emphasising a manual review rather than blindly accepting a computers best guess)

Reply:
Yeah I agree with @jaymcp and @jrmd here - Jerome's suggestion works well. Any coding standards and linting should be covered by `"My code follows the style guidelines of this project"` regardless of whether AI was used or not.